packages/run-protocol/src/proposals/econ-behaviors.js

```diff
@@ -219,9 +217,11 @@ export const setupAmm = async (
   );
 
   const chainStoragePresence = await chainStorage;
-  const ammChainStorage = await (chainStoragePresence &&
+  const storageNode = await (chainStoragePresence &&
     E(chainStoragePresence).getChildNode(AMM_STORAGE_PATH));
-  const marshaller = E(board).getPublishingMarshaller();
+  // FIXME why does this hang?
```

Comment:
@dckc any ideas?

Reply:
I think we don't resolve the `board` promise by, for example, calling `makeBoard` in any place like `setupBootstrap` (supports.js).

For an example of where we _do_ supply a board in test bootstrap, see:
https://github.com/Agoric/agoric-sdk/blob/7dbe6c0e948f0686ed77ef3439c69f6af1dc29d2/packages/run-protocol/test/test-gov-collateral.js#L175